### ISSUE TITLE
Map selection options

### ISSUE THREAD

**Issue by jongbonga** [ProteanSoftware/capacitor-start-navigation#8]
Is it possible to add a selection of maps?
e.g: I prefer to use Google map or Waze on my iOS instead of the Apple Maps

**Comment by AshleyMedway** [comment]
Are you thinking of developer picks the app to use or giving the end user a choice?

**Comment by jongbonga** [comment]
Yes @AshleyMedway , just like [this cordova version ](https://github.com/dpa99c/phonegap-launch-navigator#supported-navigation-apps)

**Comment by rvssvl** [comment]
@jongbonga  faced the same problem, need to list other available navigation maps of the user and let the user choose

**Comment by webfletch** [comment]
Any update on this at all? Currently have a Cordova App that uses the Launch Navigator allowing the user to choose which App they want to use and looking to upgrade to Capacitor so need to offer the same functionality.

**Comment by bieblebrox** [comment]
👍 

We also have a client that's asking for the option to select other navigation apps. (Same story, coming from Cordova Launch Navigator to this Capacitor plugin)

Any idea's / suggestions?

**Comment by bieblebrox** [comment]
Looks like there is already some progress: 

https://github.com/ProteanSoftware/capacitor-start-navigation/commit/e24012edebe71eeb4c4510e242a70da009b5ad13

**Comment by reinvanleirsberghe** [comment]
+1

**Comment by ckoon-infopro** [comment]
We are migrating all the Cordova plugins to Capacitor, and this is one of the features that's stopping us from going fully Capacitor. We are currently using [phonegap-launch-navigator](https://github.com/dpa99c/phonegap-launch-navigator).

**Comment by jongbonga** [comment]
> We are migrating all the Cordova plugins to Capacitor, and this is one of the features that's stopping us from going fully Capacitor. We are currently using [phonegap-launch-navigator](https://github.com/dpa99c/phonegap-launch-navigator).

I'm also using the same plugin with Capacitor just fine. just do it
